client/src/entrypoints/admin/userbar.js

```diff
@@ -261,6 +264,65 @@ class Userbar extends HTMLElement {
     function clickOutside() {
       hideUserbar();
     }
+
+    document.addEventListener('DOMContentLoaded', async () => {
+      await this.initialiseAxe();
+    });
+  }
+
+  // Initialise axe accessibility checker
+
+  async initialiseAxe() {
```

Comment:
For future refactoring:

- Explain why / how we integrate Axe
- Add a link to relevant Axe API documentation